fiatconnect-api.md

```diff
@@ -2226,12 +2226,15 @@ A KYC schema containing personal data about a user, as well as documents such as
 		postalCode?: `string`
 	},
 	phoneNumber: `string`,
+	idNumber?: `string`,
+	email?: `string`,
 	selfieDocument: `string`,
-	identificationDocument: `string`
+	identificationDocument: `string`,
+	identificationDocumentBack?: `string`,
```

Comment:
if you require `idNumber`, `email`, and `identificationDocumentBack`, then they shouldn't be optional fields. The question mark indicates that the field is optional.

Reply:
Also, I recommend that you create a new KYC schema for these fields, rather than tacking them onto an existing one. Adding a new KYC schema is a non-breaking change, since clients can incrementally add support for new schemas without breaking their existing integrations. However, changing an existing KYC schema is generally a breaking change, and something to avoid without some compelling reason for it